net/src/Sails.Remoting.Abstractions/IRemotingExtensions.cs

```diff
@@ -1,15 +1,17 @@
-﻿using Substrate.Gear.Api.Generated.Model.gprimitives;
-using System.Collections.Generic;
-using System.Threading.Tasks;
+﻿using System.Collections.Generic;
 using System.Threading;
+using System.Threading.Tasks;
 using EnsureThat;
+using Substrate.Gear.Api.Generated.Model.gprimitives;
+using GasUnit = Substrate.NetApi.Model.Types.Primitive.U64;
 using ValueUnit = Substrate.NetApi.Model.Types.Primitive.U128;
 
 namespace Sails.Remoting.Abstractions;
 
 public static class IRemotingExtensions
 {
-    public static Task<(ActorId ProgramId, byte[] EncodedReply)> ActivateAsync(
+    /// <inheritdoc cref="IRemoting.ActivateAsync(CodeId, IReadOnlyCollection{byte}, IReadOnlyCollection{byte}, GasUnit?, ValueUnit, CancellationToken)"/>
+    public static Task<Task<(ActorId ProgramId, byte[] EncodedReply)>> ActivateAsync(
```

Comment:
Suggest removing these extension methods and making methods with default values.
```
GasUnit? gasLimit = default,
ValueUnit value = ZeroValue,
CancellationToken cancellationToken = default);
```